atom/browser/font_defaults.cc

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2018 Slack Technologies, Inc.
```

Comment:
Should there be a Slack copyright here?

Reply:
I didn't realize that we need copyrights on these files.  It is fine that it has the Slack copyright as it is following the license terms.